### PR TITLE
Adding alt sequences for bwa

### PR DESCRIPTION
**Pull request by ffinfo**
Depends on https://github.com/biowdl/tasks/pull/54

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,8 @@ language: java
 script:
   - set -e
   - export CROMWELL_VERSION=34
-  - wget https://github.com/broadinstitute/cromwell/releases/download/$CROMWELL_VERSION/womtool-$CROMWELL_VERSION.jar
-  - for F in *.wdl; do echo $F; java -jar womtool-$CROMWELL_VERSION.jar validate $F; done
+#  - wget https://github.com/broadinstitute/cromwell/releases/download/$CROMWELL_VERSION/womtool-$CROMWELL_VERSION.jar
+  - wget https://barmsijs.lumc.nl/womtool-35-a7ae2d8-SNAP.jar
```
Comment (rhpvorderman) [inline .travis.yml]: UGLY :scream: 
but the only option really until cromwell-35 is released

File: align-bwamem.wdl [hunk 2]
```diff
@@ -2,7 +2,7 @@ version 1.0
 
 import "tasks/bwa.wdl" as bwa
 import "tasks/samtools.wdl" as samtools
-import "tasks/bwa.wdl" as bwa
+import "tasks/common.wdl" as common
```
Comment (rhpvorderman) [inline align-bwamem.wdl]: Maybe better to move the BWA index struct to BWA because that makes most sense.

File: environment.yml [hunk 3]
```diff
@@ -8,3 +8,4 @@ dependencies:
   - samtools==1.8
```
Comment (rhpvorderman) [inline environment.yml]: Is samtools used in this pipeline? Or does Picard do al the sorting?

File: .travis.yml (hunk 1 above)
Comment (ffinfo) [reply .travis.yml]: agree, but there no other option for now

File: environment.yml (hunk 3 above)
Comment (ffinfo) [reply environment.yml]: For star it is required. After this changes samtools is not required anymore for bwa mem

File: align-bwamem.wdl (hunk 2 above)
Comment (ffinfo) [reply align-bwamem.wdl]: BwaIndex is already inside bwa but IndexedBam is in common

File: environment.yml (hunk 3 above)
Comment (ffinfo) [reply environment.yml]: Will remove the import from the pipeline

**Pull request merged**